Gordon360/Web.config

```diff
@@ -104,7 +104,5 @@
       <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
     </providers>
   </entityFramework>
-  <connectionStrings>
-    <add name="CCTEntities1" connectionString="metadata=res://*/Models.CCT_DB_Models.csdl|res://*/Models.CCT_DB_Models.ssdl|res://*/Models.CCT_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=CCT;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" />
-  </connectionStrings>
+  <connectionStrings><add name="CCTEntities1" connectionString="metadata=res://*/Models.CCT_DB_Models.csdl|res://*/Models.CCT_DB_Models.ssdl|res://*/Models.CCT_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=CCT;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" /><add name="StudentTimesheetsEntities" connectionString="metadata=res://*/Models.StudentTimesheet_DB_Models.csdl|res://*/Models.StudentTimesheet_DB_Models.ssdl|res://*/Models.StudentTimesheet_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=StudentTimesheets;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" /></connectionStrings>
```

Comment:
Did you write that whole second <add> child, or was it autogenerated?

Reply:
It was mostly autogenerated, with the exception being that sometimes we might manually need to change the DB server address from train to prod

Reply:
Okay, cool. I'm keeping  a lookout for the code that connects a new database to the API so that I can do that for Student News.

Reply:
To connect the new database, I had to adapt the instructions [here](https://github.com/gordon-cs/gordon-360-api#updating-edmx) for updating the .edmx file. I basically replaced "CCT" with StudentTimesheets wherever that showed up in the instructions.